app/models/story.rb

```diff
@@ -198,6 +198,10 @@ class Story < ApplicationRecord
     check_tags
   end
 
+  def is_disownable_by_user?(user)
+    user && user.id == user_id
```

Comment:
does this need validate `created_at` as in [comment model](https://github.com/lobsters/lobsters/blob/master/app/models/comment.rb#L425)?

Reply:
Yes, please add a `DELETEABLE_DAYS` constant to the Story model that is twice `FLAGGABLE_DAYS` like Comment.

Reply:
it is now added.